### PR TITLE
design-add-address-interface

### PR DESCRIPTION
**Pull request by HoangVanTrinh92**

### CODE REVIEW

File: app/views/addresses/_add_form.html.erb [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-    <%= f.label :longitude %><br />
-    <%= f.text_field :lng, id: "form-lng" %>
+    <%= f.label :longitude %>
+    <%= f.text_field :lng, id: "form-lng", class: "address-input" %>
   </p>
   <p>
-    <%= f.label :capacity %><br />
-    <%= f.text_field :capacity %>
+    <%= f.label :capacity %>
+    <%= f.text_field :capacity, class: "address-input", placeholder: t("address.capacity_input") %>
```
Comment (bapboy18) [inline app/views/addresses/_add_form.html.erb]: over 80 character

File: app/assets/stylesheets/custom.css.scss [hunk 2]
```diff
@@ -11,6 +11,20 @@ h1 {
   text-align: center;
 }
 
+h3 {
+  color: #7c795d;
+  font-family: 'Trocchi', serif;
```
Comment (bapboy18) [inline app/assets/stylesheets/custom.css.scss]: used ' '

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -2,14 +2,16 @@
...
     <title><%= t "app_name" %></title>
+    <script src="https://maps.googleapis.com/maps/api/js?v=3.exp&libraries=places"></script>
     <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track" => true %>
     <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
     <%= csrf_meta_tags %>
   </head>
-  <body>
+  <body onLoad="initialize()">
     <div id="header_wrapper">
       <%= render "shared/header" %>
```
Comment (samnangcattor) [inline app/views/layouts/application.html.erb]: _header.html.erb  must be in folder  layouts

File: app/views/addresses/_add_form.html.erb [hunk 4]
```diff
@@ -1,31 +1,50 @@
...
+        <% end %>
+      </div>
+    </div>
+  </div>
+
+  <br><br>
+
+  <div id="map-add-address">
+    <div id="latlng-box">
+        <%= f.label :latitude %>
```
Comment (tranhuyhoangbka) [inline app/views/addresses/_add_form.html.erb]: indent

File: app/views/addresses/_add_form.html.erb [hunk 5]
```diff
@@ -1,31 +1,50 @@
...
+        <%= f.text_field :lat, id: "form-lat", class: "latlng-input" %>
+        <%= f.label :longitude %>
+        <%= f.text_field :lng, id: "form-lng", class: "latlng-input" %>
+    </div>
+    <input id="pac-input" class="controls" type="text">
+    <div id="map-canvas"></div>
+  </div>
+  <br>
+  <%= f.submit t("address.add_address"), class: "btn btn-primary" %></p>
```
Comment (tranhuyhoangbka) [inline app/views/addresses/_add_form.html.erb]: remove </p>

File: app/assets/javascripts/custom.js [hunk 6]
```diff
@@ -207,12 +207,34 @@ $(document).ready(function() {
...
+  var ul = $("#myTab");
+  ul.find("li.active").attr("class", "");
+
+  var li = ("<li class=\"active\"><a data-toggle=\"tab\" href=" + "#" + new_id + ">New Image</a></li>");
+  ul.append(li);
+
+  var div = $(".tab-content");
+  div.find("div.active").attr("class", "tab-pane fade");
+
+  var divTabPane = ("<div id=" + new_id + " class= \"tab-pane active\">" + content.replace(regexp, new_id) + "</div>");
```
Comment (tranhuyhoangbka) [inline app/assets/javascripts/custom.js]: > 80 char

**Pull request merged**